### PR TITLE
docs: Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

**Review by eriolchan — APPROVED** [review]

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+Incremental version for Teams Toolkit with multiple bugs fixed and the following updates：
+
+New Feature:
+- Add extension accessiblity for the disabled.
+- Add CLI command 'teamsfx preview' to preview SPFx apps.
+
+Enhancement:
+- A/B testing for Tree View design. You may randomly see either one of the two Tree View layouts in the Side bar. See below picture.
+  
+  ![TreeViewABtest](./media/TreeViewABtest.png)
```
Comment (qinezh) [inline packages/vscode-extension/CHANGELOG.md:14]: Seems the image doesn't exist.

Comment (IvanJobs) [reply packages/vscode-extension/CHANGELOG.md:14]: @qinezh  it's good from my side. Can you double confirm this from your side?

Comment (qinezh) [reply packages/vscode-extension/CHANGELOG.md:14]: @IvanJobs yeah, looks the functionality to display rich diff is not accurate... Resolved.

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/CHANGELOG.md:14]: @IvanJobs Is local resource supported here? I remember when I reference image in README.md only link in following format is supported. 
https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/media/xx.png

**Pull request merged**